README.md

```diff
@@ -25,6 +25,20 @@ Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yello
 
 <br />
 
+ - [Requirements](./README.md#requirements)
```

Comment:
The issue #3010 refers to this page:
![image](https://user-images.githubusercontent.com/11404065/118350519-7551a800-b557-11eb-8033-ba0f6a9026f9.png)

Reply:
I will add that in `SERVER.md` also.